fv3core/stencils/saturation_adjustment.py

```diff
@@ -484,10 +413,8 @@ def update_latent_heat_coefficient_i(pt1, cvm):
 @gtscript.function
 def update_latent_heat_coefficient(pt1, cvm, lv00, d0_vap):
     lhl = lv00 + d0_vap * pt1
-    lhi = LI00 + DC_ICE * pt1
     lcp2 = lhl / cvm
```

Comment:
These other two could be abstracted into a parallel function to grab lhl and lcp2

Reply:
Ok, I can create two different functions for `lhl` and `lcp2`.

Reply:
Just an idea to make the function operation depth look flatter, but not a requirement, totally up to you.

Reply:
But yes, that's what I meant